webapp/.env.example

```diff
@@ -3,6 +3,9 @@
 REACT_APP_BACKEND_URI=https://localhost:40443/
 REACT_APP_AAD_AUTHORITY=https://login.microsoftonline.com/common
 REACT_APP_AAD_CLIENT_ID=
+# Authorization scopes to access webapi when using Azure AD authorization
+# example: api://12341234-1234-1234-1234-123412341234/access_as_user
```

Comment:
nit: point to the README section you added here

Reply:
Added the readme reference and also added more detailed information for this env variable in the readme